packages/blockchain/src/utils.ts

```diff
@@ -17,3 +19,40 @@ export function parseGethGenesisState(json: any) {
   }
   return state
 }
+
+// TODO: Decide if these should be here or in Blockchain or somewhere else
```

Comment:
we can put these in block itself since nothing in them seems to pertain to blockchain

Reply:
I think you're right.  Will move them.